pygfx/objects/_events.py

```diff
@@ -331,12 +334,24 @@ def dispatch_event(self, event: Event):
 
         # Check for captured pointer events
         if pointer_id is not None and pointer_id in EventTarget.pointer_captures:
-            captured_target = EventTarget.pointer_captures.get(pointer_id)
-            # Encountered an event with pointer_id while there is a
-            # capture active, so don't bubble, and retarget the event
-            # to the captured target
-            event._retarget(captured_target)
-            event.stop_propagation()
+            captured_target_ref, event_root_ref = EventTarget.pointer_captures[
+                pointer_id
+            ]
+            captured_target, event_root = (
+                captured_target_ref(),
+                event_root_ref and event_root_ref(),
+            )
+            # If the pointer was captured in the context of another root event
+            # handler, then let's not handle this event
```

Comment:
```suggestion
            # If the pointer was captured in the context of another root event
            # handler, then let's not handle this event. It will be handled by
            # the appropriate RootEventHandler
```